### PR TITLE
Bump to core 5.3.0-alpha-1

### PR DESCRIPTION
**Pull request by annetill**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

No, just a PR that shows the impacts of breaking change about tie lines and dangling lines on open-loadflow.

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Comment by annetill** [comment]
TODO tie line contingency + check sensi

File: src/main/java/com/powsybl/openloadflow/network/impl/ContingencyTripping.java [hunk 1]
```diff
@@ -59,6 +59,23 @@ public static ContingencyTripping createBranchTripping(Network network, Branch<?
         }
     }
 
+    public static ContingencyTripping createTieLineTripping(Network network, TieLine line, String voltageLevelId) {
```
Comment (geofjamg) [inline src/main/java/com/powsybl/openloadflow/network/impl/ContingencyTripping.java:62]: It is always called with null as voltage level?

File: src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java [hunk 2]
```diff
@@ -843,15 +847,19 @@ private static LfBranch checkAndGetBranchOrLeg(Network network, String branchId,
         if (branch != null) {
             return lfNetwork.getBranchById(branchId);
         }
-        DanglingLine danglingLine = network.getDanglingLine(branchId);
-        if (danglingLine != null) {
+        DanglingLine danglingLine = network.getDanglingLine(branchId); // add unit test.
```
Comment (geofjamg) [inline src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java:850]: are we doing it?

Comment (annetill) [reply src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java:850]: yes

File: src/main/java/com/powsybl/openloadflow/network/impl/ContingencyTripping.java (hunk 1 above)
Comment (annetill) [reply src/main/java/com/powsybl/openloadflow/network/impl/ContingencyTripping.java:62]: yes, is it an issue?

**Review by geofjamg — APPROVED** [review]

**Pull request merged**